assets/queries/openAPI/2.0/schema_object_incorrect_ref/test/negative1.json

```diff
@@ -0,0 +1,62 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Simple API Overview",
+    "version": "1.0.0"
+  },
+  "paths": {
+    "/": {
+      "get": {
+        "operationId": "listVersionsv2",
+        "summary": "List API versions",
+        "responses": {
+          "200": {
+            "$ref": "#/responses/Success"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/limitParam"
+          }
+        ]
+      }
+    }
+  },
+  "responses": {
+    "Success": {
+      "description": "An array with users",
+      "schema": {
+        "$ref": "#/definitions/User"
+      }
+    }
+  },
+  "parameters": {
+    "limitParam": {
+      "name": "limit",
+      "in": "query",
```

Comment:
Same case as [here](https://github.com/Checkmarx/kics/pull/3617#discussion_r650781336). The same applies to the other samples